client/src/components/Admin/Users/User.js

```diff
@@ -67,7 +106,18 @@ User.propTypes = {
   id: PropTypes.string.isRequired,
   permissions: PropTypes.number.isRequired,
   registered: PropTypes.string.isRequired,
+  setPermissions: PropTypes.func.isRequired,
   toggleCanVote: PropTypes.func.isRequired,
 };
 
+const mapDispatchToProps = dispatch => ({
+  setPermissions: (id, canVote, permissions) => {
+    dispatch(adminSetPermissions(id, canVote, permissions));
+  },
+});
```

Comment:
For future reference:  mapDispatchToProps can be an object with actionCreators. 
Meaning that this can be written like this:

```javascript
const mapDispatchToProps = ({
  setPermissions: adminSetPermissions,
});
```